faq/prooffacehelp.php

```diff
@@ -697,6 +739,16 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 <h4>Common Tags</h4>
 
 <p>
+You may sometimes find formatting already present in the text.
+Do not add or correct this formatting information; the formatters
+will do that later in the process. However, you can remove it if
+it interferes with your proofreading. The &lt;x&gt; button in the
```

Comment:
It's not &lt;x&gt;
Rather, it is &lt;x&gt; but struckthrough, and should be shown that way.

Reply:
Good catch, fixed.